keysManagement/peerInfo.go

```diff
@@ -7,11 +7,17 @@ import (
 	crypto "github.com/ElrondNetwork/elrond-go-crypto"
 )
 
+type namedIdentity struct {
+	name     string
+	identity string
+}
+
 type peerInfo struct {
 	pid                core.PeerID
 	p2pPrivateKeyBytes []byte
 	privateKey         crypto.PrivateKey
 	machineID          string
+	namedIdentity      namedIdentity
```

Comment:
we do not need a different struct here, place directly the name & identity members